charts/aws/values.yaml

```diff
@@ -0,0 +1,102 @@
+nameOverride: ""
+fullnameOverride: ""
+labels: {}
+scraper:
+  name: aws
+  
+connectionDetials:
+  connection: ''
+  accessKey:
+    name: ''
+    value: ''
+    valueFrom:
+      serviceAccount: ''
+      helmRef:
+        key: ''
+        name: ''
+      configMapKeyRef:
+        key: ''
+        name: ''
+      secretKeyRef:
+        key: ''
+        name: ''
+
+  secretKey:
+    name: ''
+    value: ''
+    valueFrom:
+      serviceAccount: ''
+      helmRef:
+        key: ''
+        name: ''
+      configMapKeyRef:
+        key: ''
+        name: ''
+      secretKeyRef:
+        key: ''
+        name: ''
+  region: ''
+  endpoint: ''
+  skipTLSVerify: ''
+  assumeRole: ''
```

Comment:
@moshloop Should we take all the connection details as input and let the user pass what they want ?